util/topgen/merge.py

```diff
@@ -699,6 +727,9 @@ def merge_top(topcfg: OrderedDict, ipobjs: OrderedDict,
     # Add path names to declared resets
     amend_resets(gencfg)
 
+    # Combine the wakeups
```

Comment:
since i'm adding to `inter_module`, this particular amend has to be in front of `elaborate_intermodule`.  I guess `intermodule` can probably really go at the end...